workflows/images/cellranger/README.md

```diff
@@ -0,0 +1,36 @@
+This folder contains a Dockerfile for the cellranger analysis.
+
+The cellranger archive component must be downloaded separately to comply with licensing, and should be placed in this folder.
+The current version of this file is `cellranger-4.0.0.tar.gz` and can be downloaded from [10X Genomics Website](https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest) after agreeing to their license terms.
```

Comment:
This being here made me think I should download this first, but I do not need to after I read past this. 

Some headers to separate where the "instruction steps" begin would be helpful.

Reply:
You do need to download this if you want to build the docker image. But if you have access to our image stored in github, you don't need it. I will clarify.